src/common/transformations/src/transformations/common_optimizations/simplify_shape_of_sub_graph.cpp

```diff
@@ -325,17 +325,18 @@ bool ngraph::pass::SimplifyShapeOfSubGraph::run_on_model(const std::shared_ptr<n
     RUN_ON_FUNCTION_SCOPE(SimplifyShapeOfSubGraph);
     ngraph::pass::Manager manager;
     manager.set_per_pass_validation(false);
-    manager.register_pass<ngraph::pass::EliminateGatherUnsqueeze>();
-    manager.register_pass<ngraph::pass::SharedShapeOf>();
-    manager.register_pass<ngraph::pass::GroupedGatherElimination>();
+    REGISTER_PASS_SCOPE(manager, ngraph::pass, EliminateGatherUnsqueeze, )
+    REGISTER_PASS_SCOPE(manager, ngraph::pass, SharedShapeOf, _run_on_function)
+    REGISTER_PASS_SCOPE(manager, ngraph::pass, GroupedGatherElimination, )
     // GatherNopElimination depends on shape, so it requires shape propagation
     // if previous transformations has resolved some dynamic shapes.
-    manager.register_pass<ngraph::pass::Validate>();
-    manager.register_pass<ngraph::pass::GatherNopElimination>();
-    manager.register_pass<ngraph::pass::SimplifyGatherShapeOf>();
-    manager.register_pass<ngraph::pass::SimplifySecondInputOfReshape>();
+    REGISTER_PASS_SCOPE(manager, ngraph::pass, Validate, _run_on_model)
+    REGISTER_PASS_SCOPE(manager, ngraph::pass, GatherNopElimination, )
+    REGISTER_PASS_SCOPE(manager, ngraph::pass, SimplifyGatherShapeOf, )
+    REGISTER_PASS_SCOPE(manager, ngraph::pass, SimplifySecondInputOfReshape, )
+
     // TODO: potentially this Validate is not needed but it requires additional validation
-    manager.register_pass<ngraph::pass::Validate>();
+    REGISTER_PASS_SCOPE(manager, ngraph::pass, Validate, _run_on_model)
```

Comment:
Additional parameter.

Why in some places we have new additional parameters or redundant commas?